certified-connectors/Mural/apiProperties.json

```diff
@@ -0,0 +1,37 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "[[ 00000000-00000000-00000000-00000000]]",
+          "scopes": [
+            "rooms:read users:read workspaces:read murals:read murals:write identity:read templates:read"
+          ],
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "properties": {
+            "IsFirstParty": "False",
+            "IsOnbehalfofLoginSupported": false
+          },
+          "customParameters": {
+            "authorizationUrl": {
+              "value": "https://app.mural.co/api/public/v1/authorization/oauth2/"
+            },
+            "tokenUrl": {
+              "value": "https://app.mural.co/api/public/v1/authorization/oauth2/token"
+            },
+            "refreshUrl": {
+              "value": "https://app.mural.co/api/public/v1/authorization/oauth2/refresh"
+            }
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#FFFFFF",
```

Comment:
@dmytroIvanovDev You have agreed to not have white color as iconBrandColor for your connector icon, but you still have provided white color. Please update it.

![image](https://user-images.githubusercontent.com/56276394/149010524-75a3262d-c650-49b5-ad17-8664e87e1847.png)